src/woo_publications/typing.py

```diff
@@ -1,3 +1,7 @@
+from typing import TypedDict
+
 type JSONPrimitive = str | int | float | bool | None
 type JSONValue = JSONPrimitive | JSONObject | list[JSONValue]
 type JSONObject = dict[str, JSONValue]
+
+ApiUser = TypedDict("ApiUser", {"user_id": str, "user_repr": str})
```

Comment:
why define a new type for what is essentially https://github.com/GeneriekPublicatiePlatformWoo/registratie-component/blob/main/src/woo_publications/logging/typing.py#L4 ? I would just make sure you use `ActingUser` everywhere.